lib/archethic/replication/transaction_validator.ex

```diff
@@ -349,10 +339,15 @@ defmodule Archethic.Replication.TransactionValidator do
     %LedgerOperations{unspent_outputs: expected_unspent_outputs} =
       %LedgerOperations{
         fee: fee,
-        transaction_movements: transaction_movements
+        transaction_movements: transaction_movements,
+        tokens_to_mint: LedgerOperations.get_utxos_from_transaction(tx, timestamp)
       }
-      |> LedgerOperations.from_transaction(tx, tx.validation_stamp.timestamp)
-      |> LedgerOperations.consume_inputs(tx.address, inputs, tx.validation_stamp.timestamp)
+      |> LedgerOperations.consume_inputs(
+        address,
+        inputs,
+        timestamp
+      )
+      |> elem(1)
```

Comment:
Here you should keep the `enough_funds?` boolean and return an error of the user don't have enough funds for its transfers and return `insufficient_funds` as it was done before